src/test/java/WebDriverSeleniumTests/googleSearchForWikiChrome.java

```diff
@@ -0,0 +1,93 @@
+package WebDriverSeleniumTests;
+
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Logger;
+import ru.yandex.qatools.ashot.AShot;
+import ru.yandex.qatools.ashot.Screenshot;
+import ru.yandex.qatools.ashot.shooting.ShootingStrategies;
+
+public class googleSearchForWikiChrome {
+
+    @Test
+    public void GoogleSearchForWikiIsNotEmpty() throws IOException {
+
+        final Logger logger = Logger.getGlobal();
+
+        System.setProperty("webdriver.edge.driver", "src/test/resources/chromedriver.exe");
+
+        ChromeOptions chromeoptions = new ChromeOptions();
+        chromeoptions.addArguments("–lang= en");
+
+        WebDriver driver = new ChromeDriver(chromeoptions);
+
+        driver.get("http://google.com");
+        new WebDriverWait(driver, 10).until(ExpectedConditions.presenceOfElementLocated(By.name("q")));
+        driver.manage().window().maximize();
+
+        WebElement googleSearchField = new WebDriverWait(driver, 10).until(ExpectedConditions.presenceOfElementLocated(By.name("q")));
+        googleSearchField.sendKeys("wikipedia");
+
+        List<WebElement> googleSearchButton = driver.findElements(By.name("btnK"));
```

Comment:
Why you use List for one button? And why findElementS ? As far as I see - there is only 1 search button?

Reply:
Just tried how the list method works  - findElement could be used, for sure

Reply:
In such case - it is ok